runtimes/eden/src/lib.rs

```diff
@@ -83,13 +83,14 @@ construct_runtime! {
 
         // Governance
         Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 10,
-        CompanyReserve: pallet_reserve::<Instance1> = 11,
-        InternationalReserve: pallet_reserve::<Instance2> = 12,
-        UsaReserve: pallet_reserve::<Instance3> = 13,
-        Vesting: pallet_grants = 14,
-        Mandate: pallet_mandate::{Pallet, Call, Storage, Event<T>} = 15,
-        TechnicalCommittee: pallet_collective::<Instance1> = 16,
-        TechnicalMembership: pallet_membership::<Instance3> = 17,
+        Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 11,
+        CompanyReserve: pallet_reserve::<Instance1> = 12,
+        InternationalReserve: pallet_reserve::<Instance2> = 13,
+        UsaReserve: pallet_reserve::<Instance3> = 14,
+        Vesting: pallet_grants = 15,
+        Mandate: pallet_mandate::{Pallet, Call, Storage, Event<T>} = 16,
+        TechnicalCommittee: pallet_collective::<Instance1> = 17,
+        TechnicalMembership: pallet_membership::<Instance3> = 18,
```

Comment:
You shouldn't change the pallet indexes as it would break compatibility with hardware wallets.

Reply:
Sure will refactor it

Reply:
Done